assets/css/_circle_button.scss

```
@@ -0,0 +1,29 @@
+@use "./color/tokens_2024" as tokens;
+
+.c-circle-button {
+  width: 2rem;
+  height: 2rem;
+
+  background-color: tokens.$gray-50;
+  border-radius: 50%;
+
+  &:hover {
+    background-color: tokens.$gray-100;
+  }
+
+  &:focus-visible {
+    border: 0.0625rem solid var(--bs-primary);
+
+    .c-circle-button__content {
+      margin: calc(0.25rem - 0.0625rem);
+    }
+  }
+
+  &--active {
```

```suggestion
  &.c-circle-button--active {
```

When there are conflicting rules, like `:hover` versus a specific class, CSS will pick whichever rule trigger is more specific. So without applying this, the rule trigger was `.c-circle-button:hover` (class plus hover state) versus `.c-circle-button--active` (just a class). With applying this change, now it would be `.c-circle-button .c-circle-button--active` (two classes), which I guess CSS sees as more specific than `.c-circle-button:hover`

---

lol great minds